src/rctmon/device_manager.py

```diff
@@ -436,6 +436,22 @@ def _cb_solar_generator(self, oid: int, value: Union[float, bool]) -> None:
         except TypeError:
             log.warning('Got wrong type %s for %s', type(value), R.get_by_id(oid).name)
 
+    def _cb_sensors(self, oid: int, value: Union[float, bool]) -> None:
+        '''
+        Callback for storing sensors information.
+        '''
+        try:
+            if oid == 0xF79D41D9:
+                self.readings.temperature_heatsink = ensure_type(value, float)
+            elif oid == 0x4F735D10:
+                self.readings.temperature_heatsink_batt = ensure_type(value, float)
+            elif oid == 0xC24E85D0:
+                self.readings.temperature_core = ensure_type(value, float)
+            elif oid == 0x902AFAFB:
+                self.readings.temperature_battery = ensure_type(value, float)
```

Comment:
As written above, this is already working in the battery manager. The fact that `Readings` has a `temperature_battery` field is a leftover from an earlier version before I separated the battery-related parts from the device manager and into the battery manager in an attempt to reduce the size of the file.

Reply:
Thanks, I removed it from sensors callback and add_ids